README.md

```diff
@@ -498,3 +500,9 @@ If you want to clean artifacts of all contexts, you should to run `clean --curre
 * `eden utils gcp` - sub-commands to work with Google Cloud Platform
 * `eden utils export` - sub-command to save certs and configs into tar.gz
 * `eden utils import` - sub-commands to load certs and configs into tar.gz
+
+## Parallels
+
+0. Shut down the virtual machine before changing the hypervisor type.
```

Comment:
Wait, wait -- where's the virtual machine coming from? Seems like some steps are missing right before this one.